src/test/java/uk/gov/hmcts/cmc/claimstore/controllers/CaseMetadataControllerTest.java

```diff
@@ -0,0 +1,146 @@
+package uk.gov.hmcts.cmc.claimstore.controllers;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.cmc.claimstore.controllers.support.CaseMetadataController;
+import uk.gov.hmcts.cmc.claimstore.exceptions.NotFoundException;
+import uk.gov.hmcts.cmc.claimstore.services.ClaimService;
+import uk.gov.hmcts.cmc.claimstore.services.UserService;
+import uk.gov.hmcts.cmc.claimstore.services.notifications.fixtures.SampleUser;
+import uk.gov.hmcts.cmc.domain.models.CaseMetadata;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Collections.singletonList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CaseMetadataControllerTest {
```

Comment:
I would recommend converting this to an api or functional test, it doesn't serve much value imo

Reply:
Added api tests - want me to remove this as a target of unnecessary maintenance or just keep it for the coverage stats?

Reply:
api tests should already be included in coverage